tests/std/tests/P0896R4_views_reverse/test.cpp

```diff
@@ -230,6 +230,63 @@ constexpr bool test_one(Rng&& rng, Expected&& expected) {
         }
     }
 
+#if _HAS_CXX23
+    // Validate view_interface::cbegin

```

Comment:
Should we additionally have a "Ditto" comment (as in the filter view test) explaining the repeated calls below, as the comment on line 173 does?

Reply:
I've added this comment (https://github.com/microsoft/STL/pull/3612/commits/69b21c70a5439ae63743d92bec3418b5207c93de).